src/components/inputs/editor-input-v3.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef, useMemo } from "react";
+import JoditEditor from "jodit-react";
+
+const TextEditorV3 = ({ id, value, error, className, onChange, options = {} }) => {
+    const editor = useRef(null);
+
+    // all options from https://xdsoft.net/jodit/docs/,
+    const config = useMemo(
+        () => ({
+            placeholder: "Start typings...",
+            className,
+            buttons: ["bold", "italic", "strikethrough", "underline", "|", "source", "|", "font", "fontsize","align", "|", "ul", "ol", "image", "link", "|", "undo", "redo"],
```

Comment:
could we make this buttons configurable by prop and have this as default value prop ?